compiler/pipes/calc-empty-functions.cpp

```diff
@@ -49,6 +48,7 @@ FunctionData::body_value get_vertex_body_type(VertexPtr vertex) {
     case op_seq:
       return calc_seq_body_type(vertex.as<op_seq>());
     case op_empty:
+    case op_global:
```

Comment:
Why do we need it?

Reply:
It was just missing. Such a file:
```
global $a;
function f() { }
```
was earlier considered as non-empty.